guide/inheritance-wallet/landing-page.md

```diff
@@ -0,0 +1,167 @@
+   alt-text = "An image showing two parents handing bitcoin to their two children."
+   width = 1600
+   height = 600
+   layout = "full-width"
+%}
+
+# Family inheritance wallet 
+{: .no_toc }
+
+The family inheritance wallet is an app designed to create flexible wallets for family savings. It offers features designed to help with sustainable long-term storage and inheritance planning.
+
+But when it comes to inheritance, technology is not everything. Knowledge, documentation, practice  and social  processes are important for any setup to work over the long term. This exploration includes not only designs and prototypes of the wallet application, but also thoughts around social processes to ensure that funds will not be lost.
+
+## What are we building?
+
+Imagine a product that helps you create and use a custody setup for your bitcoin savings while including your loved ones from the beginning. But because you plan to live a long and happy life, the app allows you to set up a recovery path that helps you recover your funds even in catastrophic scenarios. 
+
+On top of that, the app allows you to add your heirs' hardware wallets and use them to create a dedicated inheritance key set. Since it is designed for long-term storage, it also includes regular key checks and other supporting features. 
+
+All of this is done without compromising your financial privacy. You don't want anyone to know how much bitcoin you have or monitor your transactions.
+
+## Chapters
+
+### [Use case & scope]({{ '/guide/inheritance-wallet/introduction/' | relative_url }})
+
+<div class="center" markdown="1">
+
+{% include image.html
+   image = "assets/images/guide/inheritance-wallet/icon-introduction.png"
+   retina = "assets/images/guide/inheritance-wallet/icon-introduction@2x.png"
+   alt-text = ""
+   width = 80
+   height = 80
+   layout = "float-left"
+   link-url = "/guide/inheritance-wallet/introduction/"
+%}
+
+Before we dive into the details, we will introduce you to the Jones family. We are also going to look at how they plan set up their family savings in general. 
+
+</div>
+
+---
+
+### [Wallet creation]({{ '/guide/inheritance-wallet/wallet-creation/' | relative_url }})
+
+<div class="center" markdown="1">
+
+{% include image.html
+   image = "assets/images/guide/inheritance-wallet/icon-wallet-creation.png"
+   retina = "assets/images/guide/inheritance-wallet/icon-wallet-creation@2x.png"
+   alt-text = ""
+   width = 80
+   height = 80
+   layout = "float-left"
+   link-url = "/guide/inheritance-wallet/wallet-creation/"
+%}
+
+Our application lets users create flexible multi-key wallets with timelocked recovery paths to help them recover funds while they are still alive. On top of that, users can create and manage dedicated inheritance keys that are held by their heirs.
+
+</div>
+
+---
+
+### [Wallet backup]({{ '/guide/inheritance-wallet/backup/' | relative_url }})
+
+<div class="center" markdown="1">
+
+{% include image.html
```

Comment:
This image does not seem to show up on my end. 
![Screenshot 2024-05-13 101624](https://github.com/BitcoinDesign/Guide/assets/90313394/b94a2188-7829-4979-8751-8f655181891a)